src/app/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import type { Meta, Story } from '@storybook/react';
+import React from 'react';
+import { COLORS } from '../../lib/colors';
+import { ICONS } from '../Icon/icons';
+import type { ButtonProps } from './Button';
+import Button from './Button';
+
+export default {
+  title: 'Components/Button',
+  component: Button,
+  argTypes: {
+    icon: {
+      options: [``, ...Object.keys(ICONS)],
```

Comment:
```suggestion
      options: Object.keys(ICONS),
```
Wouldn't this be sufficient?

Reply:
"No icon" is a relevant option here, too

Reply:
I changed the Icon component and added a "none" key